### PR TITLE
Replace example for string.normalize()

### PR DESCRIPTION
**Pull request by wbamberg**
As part of https://github.com/mdn/sprints/issues/2594 I decided that the interactive example for [`normalize()`](https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/normalize ) is not very readable: these long lines assembling complex strings are hard to follow.

So this PR rewrites it. It only covers "NFD" and "NFC", but is already as long as I'd like an example to be.

### CODE REVIEW

**Comment by wbamberg** [comment]
@chrisdavidmills , please could you review this when you get the time?

**Comment by wbamberg** [comment]
> So, I'm approving this, but I do wonder whether this is a bit too long to be an ie at the top of the page? Couldn't you just remove the last bit that uses NFD, and just stick with the NFC stuff? I feel that this on its own shows a good usage of normalize(), and your in-page examples that I reviewed previously expand on it nicely and show the difference between NFC and NFD.

I think showing just NFC is a good idea, just to make the example look simpler and less intimidating. But unfortunately even just showing NFC is 15 lines, which is just over the limit for a standard-size editor. And it's hard to think of how to compress it further without losing useful information. So removing NFD won't help with the sizing issue.

> If that isn't something you want to do, perhaps you could show a smaller code window and just accept that people will have to scroll to see all the code?

This is going to be tricky since https://github.com/mdn/bob/pull/368 landed, because that automatically selects the taller window based on the actual example size.

So options include:

1. do nothing

2. update https://github.com/mdn/bob/pull/368 so we can manually set code window size, as we used to, perhaps to override the automatically selected one

3. ditch the whole idea of "taller" and just use a single size, and let it overflow

Since (2) is extra work (and more to maintain, and more for contributors to understand) then we should only do it if *some* longer examples should be allowed to overflow, while others should get a taller window. I'm not sure how we should make this determination really. For example, why should this example be allowed to overflow, but [`Proxy.handler.preventExtensions()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy/handler/preventExtensions) should not?

Which would leave (3), but my belief, based the way I saw people interacting with these things in user testing, is that users you need to scroll to see some of the output, then almost noone will do it.

As a possible consolation, there aren't very many examples that overflow. This analysis: https://github.com/mdn/interactive-examples/issues/464#issuecomment-360572522 is old but still probably roughly accurate (I'm going to repeat it after https://github.com/mdn/bob/pull/368 permeates) and that indicated that only 24 examples had a length > 13, which gets you the taller editor.

**Comment by wbamberg** [comment]
I've removed the "NFD" variant, for simplicity, and also simplified the output strings, as without NFD there didn't seem much value in listing the exact string lengths.

**Comment by chrisdavidmills** [comment]
That looks better to me Will. About the concerns I raised earlier, I think the best thing for now is the "do nothing" option, as this isn't broken by any means, and we have higher priority things for you to be working on right now.

**Comment by wbamberg** [comment]
Thanks Chris!
